exonum/src/helpers/fabric/builder.rs

```diff
@@ -139,6 +136,31 @@ impl NodeBuilder {
         .map(|c| (c.name(), CollectedCommand::new(c)))
         .collect()
     }
+
+    fn read_run_context(self, ctx: &Context) -> Node {
+        let config_file_path = ctx
+            .get(keys::NODE_CONFIG_PATH)
+            .expect("Could not find node_config_path");
+        let config = ctx
+            .get(keys::NODE_CONFIG)
+            .expect("could not find node_config");
+        let db = Run::db_helper(ctx, &config.database);
+        let services: Vec<Box<dyn Service>> = self
+            .service_factories
+            .into_iter()
+            .map(|mut factory| factory.make_service(ctx))
+            .collect();
+
+        let consensus_passphrase = env::var(EXONUM_CONSENSUS_PASS).unwrap_or_default();
```

Comment:
Am I reading this correctly that the password can only be input from env when starting a node? This doesn't sound right.

Reply:
As far as I know, that's what on we agreed for the first iteration.